projects/ADuCM3029_demo_cn0326/src/AD7793.c

```diff
@@ -176,7 +176,7 @@ uint32_t AD7793_Scan(enMode mode,  uint8_t ui8channel)
 
 		//DioClr(CS_PORT, CS_PIN);
 
-		while ((AD7793_ReadRegister(AD7793_REG_STAT)& RDY_BIT) != RDY_BIT);
+	while ((AD7793_ReadRegister(AD7793_REG_STAT)& RDY_BIT) == RDY_BIT);
 
 	ui32result = AD7793_ReadRegister(AD7793_REG_DATA);
 
```

Comment:
there is no need to store the result in a new variable, we can directly return here:
`return AD7793_ReadRegister(AD7793_REG_DATA);`